src/skycoin/config.go

```diff
@@ -391,6 +391,7 @@ func (n *NodeConfig) applyConfigMode(configMode string) {
 	}
 }
 
+// nolint
```

Comment:
Does it allow us to specify which linter to disable for this line?

Reply:
With gometalinter, we would do something like `nolint: unparam`.

Also, gometalinter allows us to emit a warning if a `nolint` directive is unnecessary. If this option is available in the new method, use it.

Reply:
@gz-c yes, you can use the same also as gometalinter

Reply:
Does it have an option equivalent to `--warn-unmatched-nolint`?

Reply:
That does not need it, because if the directive you declared is not valid, simply omit the declaration `// nolint directive`.

Reply:
What is the directive for this `// nolint` clause?